frontend/src/old-pages/Clusters/Instances.tsx

```diff
@@ -42,77 +42,94 @@ import {InstanceStatusIndicator} from '../../components/Status'
 import EmptyState from '../../components/EmptyState'
 import DateView from '../../components/date/DateView'
 
-function InstanceActions({
-  fleetStatus,
-  instance,
-}: {
-  fleetStatus: ComputeFleetStatus
-  instance: Instance
-}) {
-  const pending = useState(['app', 'clusters', 'action', 'pending'])
-  const clusterName = useState(['app', 'clusters', 'selected'])
-  const navigate = useNavigate()
-  const logHref = `/clusters/${clusterName}/logs?instance=${instance.instanceId}`
+function InstanceActions({instance}: {instance?: Instance}) {
   const {t} = useTranslation()
+  const navigate = useNavigate()
+
+  const clusterName = useState(['app', 'clusters', 'selected'])
+  const fleetStatus: ComputeFleetStatus = useState([
+    'clusters',
+    'index',
+    clusterName,
+    'computeFleetStatus',
+  ])
+  const logHref = `/clusters/${clusterName}/logs?instance=${
+    instance && instance.instanceId
```

Comment:
suggestion: even though it is not a problem in this specific case, I'd avoid building links with `null` inside. This evaluates to the string `"null"` when `instance` is null.

A bit safer could be 

```
instance ? `/cluster/...${instance.instanceId}` : ''`
```